src/screens/ReceiveCoinsScreen.tsx

```diff
@@ -0,0 +1,188 @@
+import bip21 from 'bip21';
+import React, { Component } from 'react';
+import { View, StyleSheet, Text, InteractionManager } from 'react-native';
+import QRCode from 'react-native-qrcode-svg';
+import Share from 'react-native-share';
+import { NavigationScreenProps, NavigationInjectedProps } from 'react-navigation';
+
+import { Header, ScreenTemplate, Button } from 'app/components';
+import { Transaction, Route } from 'app/consts';
+import i18n from 'app/locale';
+import { typography, palette } from 'app/styles';
+
+import BlueApp from '../../BlueApp';
+import { Chain } from '../../models/bitcoinUnits';
+
+type Props = NavigationInjectedProps;
```

Comment:
with `NavigationInjectedProps` generic param you can define what params actually are
```
NavigationInjectedProps<{ secret: string }>
```